src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -124,7 +129,7 @@ public function registerContainerConfiguration(LoaderInterface $loader)
             $kernelDefinition->addTag('routing.route_loader');
 
             $container->addObjectResource($this);
-            $container->fileExists($this->getProjectDir().'/config/bundles.php');
```

Comment:
There are other occurrences in `FrameworkBundle/DependencyInjection/FrameworkExtension.php` ( `<project_dir>/config/validator` and `<project_dir>/config/serializer` ).

Also, we are hardcoding it here:
https://github.com/symfony/symfony/blob/46b2be98e88a05191181199bd514263210b7ab03/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L163